### PR TITLE
Enable win to use shell script

### PR DESCRIPTION
**Pull request by llxia**
- remove WORKSPACE key word as it does not work in sh on wins
- remove maketest.bat and get.bat

Issues: #407

Signed-off-by: lanxia <lan_xia@ca.ibm.com>

### CODE REVIEW

**Comment by llxia** [comment]
Tests:
Regular Win build: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/TestBuild_Sandbox/476/
Grinder (Win): https://ci.adoptopenjdk.net/view/work%20in%20progress/job/TestBuild_Sandbox3/69/
Grinder (linux): https://ci.adoptopenjdk.net/view/work%20in%20progress/job/TestBuild_Sandbox3/68/

**Comment by llxia** [comment]
fyi @jdekonin @AdamBrousseau

File: buildenv/jenkins/JenkinsfileBase [hunk 1]
```diff
@@ -137,18 +137,17 @@ def setup() {
...
-                }
-            } else {
-                if ( SPEC.contains('win') ) {
-                    bat "$WORKSPACE/openjdk-tests/get.bat -sdkdir $WORKSPACE -testdir $WORKSPACE/openjdk-tests -platform $PLATFORM -jvmversion $JVM_VERSION -sdk_resource $SDK_RESOURCE ${CUSTOMIZED_SDK_URL_OPTION} ${OPENJ9_REPO_OPTION} ${OPENJ9_BRANCH_OPTION} ${OPENJ9_SHA_OPTION}"
-                } else {
-                    sh "$WORKSPACE/openjdk-tests/get.sh -s $WORKSPACE -t $WORKSPACE/openjdk-tests -p $PLATFORM -v $JVM_VERSION -r $SDK_RESOURCE ${CUSTOMIZED_SDK_URL_OPTION} ${OPENJ9_REPO_OPTION} ${OPENJ9_BRANCH_OPTION} ${OPENJ9_SHA_OPTION} ${VENDOR_TEST_REPOS} ${VENDOR_TEST_BRANCHES} ${VENDOR_TEST_DIRS} ${VENDOR_TEST_SHAS}"
-                }
-            }
+			dir( WORKSPACE) {
+				if (params.USER_CREDENTIALS_ID) {
```
Comment (jdekonin) [inline buildenv/jenkins/JenkinsfileBase]: You can eliminate the if check with an addition of `, ignoreMissing: true`.

`sshagent(credentials:["${params.USER_CREDENTIALS_ID}"], ignoreMissing: true)`

File: buildenv/jenkins/JenkinsfileBase [hunk 2]
```diff
@@ -172,17 +171,14 @@ def buildTest() {
 			} catch (Exception e) {
 				echo 'Cannot run copyArtifacts from test.getDependency. Skipping copyArtifacts...'
 			}
-
+			
+			MK_TEST_CMD = "./openjdk-tests/maketest.sh ./openjdk-tests"
 			if( params.JCK_GIT_REPO ) {
```
Comment (jdekonin) [inline buildenv/jenkins/JenkinsfileBase]: You can eliminate the if check with an addition of , ignoreMissing: true.

File: buildenv/jenkins/JenkinsfileBase (hunk 1 above)
Comment (llxia) [reply buildenv/jenkins/JenkinsfileBase]: Changeset is updated. Thanks

**Comment by llxia** [comment]
- re-tested with new changeset: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/TestBuild_Sandbox3/70/
- Also tested with JCK

**Comment by llxia** [comment]
@ShelleyLambert Could you please disable `test-packet-x64-windows-2012r2-1` in AdoptOpenJDK Jenkins? Thanks.

This PR works on newly enabled wins `test-azure-win2012r2-x64-1`.
For more detail, please see https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/378

**Comment by smlambert** [comment]
I have temporarily disabled test-packet-x64-windows-2012r2-1 with a link to issue 407

**Comment by llxia** [comment]
Thanks Shelley.
Merge the PR.

**Pull request merged**